In [1]:
import json
import requests
import pandas as pd
import matplotlib.pyplot as plt
import seaborn as sns
import numpy


# Testing Sites


In [2]:
testing_sites_df =  pd.read_csv('https://raw.githubusercontent.com/dcrago/Team50_Project/main/COVID-19%20Testing%20Locations%20Symbolized%20by%20Test%20Type.csv?token=ARUSVWL2E75VLQXSQ5HVWPTAA6EAG')

In [ ]:
testing_sites_df.drop(columns = ['Owner Name','Phone','Website','Operational Hours','Vehicle Capacity','CreationDate','Drive-through','Appointment Only','Referral Required','Call first','Virtual/Telehealth Screening','Local Health Department URL','Vehicle Required','Offering same-day diagnostic (not antibody) results','Offering Take-Home Test','Status Color'], inplace = True)
testing_sites_df.drop(columns = ['Public Form Submission','Minimum Age','Wheelchair accessibility ','Accessibility Support'], inplace = True)

In [ ]:

testing_sites_df['county']= testing_sites_df['County']
testing_sites_df
testing_sites_df.sort_values(by ='County')


In [ ]:
testing_sites_df['county_states']= testing_sites_df['county'] + ' , ' + testing_sites_df['State or Territory']
testing_sites_df

In [ ]:
_ = testing_sites_df['county_states'].unique
_

# Covid

In [2]:
Covid_data = pd.read_csv("https://raw.githubusercontent.com/nytimes/covid-19-data/master/us-counties.csv")

In [3]:
Covid_data['mortality'] = Covid_data['deaths']/Covid_data['cases']
Covid_data.tail()

date      county    state     fips  cases  deaths  mortality
972436  2021-01-27  Sweetwater  Wyoming  56037.0   3428    32.0   0.009335
972437  2021-01-27       Teton  Wyoming  56039.0   3020     6.0   0.001987
972438  2021-01-27       Uinta  Wyoming  56041.0   1948    12.0   0.006160
972439  2021-01-27    Washakie  Wyoming  56043.0    863    25.0   0.028969
972440  2021-01-27      Weston  Wyoming  56045.0    610     4.0   0.006557

# Merge covid testing sites

In [6]:
testing_sites_covid = Covid_data.merge(testing_sites_df,
    how='inner',
    on = 'county'
)

testing_sites_covid

NameError: name 'testing_sites_df' is not defined

In [7]:
mort_county.corr()[['asthma_value_times_count', 'confirmed_deaths']]

NameError: name 'mort_county' is not defined

# Asthma

In [4]:
asthma = pd.read_csv('500_Cities__Current_asthma_among_adults_aged___18_years.csv')

In [5]:
asthma['state'] = asthma['StateDesc']
asthma['asthma_value_times_count'] = (asthma['Data_Value'] * asthma['PopulationCount'])/100
asthma['asthma_value_times_count']

0         528.612
1        5524.416
2         478.986
3         359.048
4         244.664
           ...   
29001     403.506
29002      45.235
29003     135.013
29004     137.800
29005     363.792
Name: asthma_value_times_count, Length: 29006, dtype: float64

# asthma.drop(columns = ['Category','UniqueID','Measure','Short_Question_Text','Data_Value_Footnote','CategoryID','MeasureId','CityFIPS','TractFIPS'], inplace = True)
asthma.columns

In [8]:
from IPython.display import HTML

In [9]:
'''
changed aggfunc to mean because were dealing with percentage value, 
sum misrepresented the data
'''

asthma1 = asthma.groupby(['state']).mean()
asthma_state_count = asthma1['asthma_value_times_count']
asthma1

Year  Data_Value  Low_Confidence_Limit  \
state                                                     
Alabama        2017.0   11.425490             10.760224   
Alaska         2017.0    9.040351              8.621053   
Arizona        2017.0   10.344531              9.794141   
Arkansas       2017.0    9.733594              9.216406   
California     2017.0    8.897829              8.427422   
Colorado       2017.0    9.454213              8.948455   
Connecticut    2017.0   10.980085             10.433051   
Delaware       2017.0   12.065385             11.392308   
District of C  2017.0   10.266111              9.639444   
Florida        2017.0    8.749375              8.273769   
Georgia        2017.0    9.877253              9.292918   
Hawaii         2017.0    8.848729              8.496186   
Idaho          2017.0    9.471233              8.954795   
Illinois       2017.0    9.436790              8.872067   
Indiana        2017.0   11.068843             10.452612   
Iowa           2017.0    9.690594              9.169802   
Kansas         2017.0    9.502903              9.017097   
Kentucky       2017.0   10.698120             10.112406   
Louisiana      2017.0    9.966142              9.334908   
Maine          2017.0   10.359091              9.813636   
Maryland       2017.0   11.478109             10.761692   
Massachusetts  2017.0   11.256863             10.662157   
Michigan       2017.0   12.239153             11.531481   
Minnesota      2017.0    8.173457              7.733642   
Mississippi    2017.0   11.103797             10.379747   
Missouri       2017.0    9.904740              9.343567   
Montana        2017.0    9.414894              8.887234   
Nebraska       2017.0    8.663761              8.178899   
Nevada         2017.0   10.050678              9.542818   
New Hampshire  2017.0   11.100000             10.482353   
New Jersey     2017.0    9.969255              9.408075   
New Mexico     2017.0    9.806542              9.310748   
New York       2017.0   10.044032              9.471895   
North Carolin  2017.0    9.820833              9.283460   
North Dakota   2017.0    8.533333              8.003704   
Ohio           2017.0   11.116329             10.480691   
Oklahoma       2017.0   10.470190              9.941015   
Oregon         2017.0   10.383523              9.838068   
Pennsylvania   2017.0   11.165581             10.529302   
Rhode Island   2017.0   12.115094             11.507547   
South Carolin  2017.0    9.938418              9.330508   
South Dakota   2017.0    8.538333              8.070000   
Tennessee      2017.0   10.802294             10.161185   
Texas          2017.0    8.508445              8.013846   
United States  2017.0    8.950000              8.800000   
Utah           2017.0    9.296813              8.775299   
Vermont        2017.0   11.153846             10.338462   
Virginia       2017.0   10.224376              9.683877   
Washington     2017.0   10.088136              9.603766   
West Virginia  2017.0   11.266667             10.700000   
Wisconsin      2017.0   10.694203             10.070773   
Wyoming        2017.0    9.644444              9.155556   

               High_Confidence_Limit  PopulationCount  \
state                                                   
Alabama                    12.210084     7.620821e+03   
Alaska                      9.522807     1.535926e+04   
Arizona                    10.979785     1.099551e+04   
Arkansas                   10.329687     1.114484e+04   
California                  9.427933     1.196123e+04   
Colorado                   10.052247     1.023229e+04   
Connecticut                11.602542     1.107831e+04   
Delaware                   12.830769     8.175115e+03   
District of C              11.047222     9.973309e+03   
Florida                     9.306025     1.109481e+04   
Georgia                    10.566738     9.821329e+03   
Hawaii                      9.244492     1.186565e+04   
Idaho                      10.07

In [29]:
#putting years in 1 column before merge
#didn't use this df because we have another thats grouped by state
asthma_premerge = asthma[['StateAbbr','Year','Data_Value']]


# Census -people with insurance

In [10]:

census_insurance = pd.read_csv('https://www2.census.gov/programs-surveys/demo/datasets/income-poverty/time-series/data-extracts/2018/cps-asec-bridge-file/hhpub18.csv')
census_insurance.columns
#HCOV NOW_HCOV 
# HTML(filename='www.census.gov/data/tables/time-series/demo/health-insurance/historical-series/hib.html')

Index(['H_IDNUM', 'GEREG', 'GESTFIPS', 'GEDIV', 'HRHTYPE', 'HEFAMINC',
       'H_MONTH', 'H_YEAR', 'H_TENURE', 'H_HHNUM',
       ...
       'THCHCARE_VAL', 'THPROP_VAL', 'GTCBSA', 'GTCO', 'GTCBSAST', 'GTCBSASZ',
       'GTCSA', 'GTMETSTA', 'GTINDVPC', 'MMYY'],
      dtype='object', length=134)

# Merging covid and asthma data

In [11]:
# Covid_data & asthma
var_of_interest= ['confirmed_deaths','confirmed_cases', ]

In [12]:
covid_asthma = pd.merge(asthma1,Covid_data,
    how="left",                      
    left_on = ['state'],
    right_on = ['state']) 
# covid_asthma.drop('United States', axis = 0)
covasdrop= ['fips','Low_Confidence_Limit','High_Confidence_Limit','Year','asthma_value_times_count','PopulationCount']
covid_asthma = covid_asthma.drop(columns = covasdrop)
covid_asthma = covid_asthma.rename(columns={'Data_Value':'perc pop with asthma','mortality':'covid mortality'})

In [13]:
covid_asthma

state  perc pop with asthma        date      county   cases  deaths  \
0       Alabama             11.425490  2020-03-13      Elmore     1.0     0.0   
1       Alabama             11.425490  2020-03-13   Jefferson     2.0     0.0   
2       Alabama             11.425490  2020-03-13   Limestone     1.0     0.0   
3       Alabama             11.425490  2020-03-13  Montgomery     1.0     0.0   
4       Alabama             11.425490  2020-03-13  Tuscaloosa     1.0     0.0   
...         ...                   ...         ...         ...     ...     ...   
903908  Wyoming              9.644444  2021-01-27  Sweetwater  3428.0    32.0   
903909  Wyoming              9.644444  2021-01-27       Teton  3020.0     6.0   
903910  Wyoming              9.644444  2021-01-27       Uinta  1948.0    12.0   
903911  Wyoming              9.644444  2021-01-27    Washakie   863.0    25.0   
903912  Wyoming              9.644444  2021-01-27      Weston   610.0     4.0   

        covid mortality  
0              0.000000  
1              0.000000  
2              0.000000  
3              0.000000  
4              0.000000  
...                 ...  
903908         0.009335  
903909         0.001987  
903910         0.006160  
903911         0.028969  
903912         0.006557  

[903913 rows x 7 columns]

In [14]:
covid_asthma.corr()[['perc pop with asthma', 'deaths']]

perc pop with asthma    deaths
perc pop with asthma              1.000000  0.013371
cases                            -0.019744  0.735856
deaths                            0.013371  1.000000
covid mortality                   0.027450  0.013347

# AQI

In [25]:
# #not updated through 12/2020
# with open('waqi-covid19-airqualitydata-2020.csv') as AQI:
#     AQI_older = pd.read_csv(AQI)
# AQI_older.head()


In [24]:
# with open('waqi-covid19-airqualitydata-2020.csv') as AQI:
#     AQI_old = pd.read_csv(AQI)
# AQI_old.head()


In [16]:
#this takes us to a document where we need to delete the first five rows bc it's just information 
#updated currently woohoo!
AQI = pd.read_excel('AQI_US.xlsx')
AQI.head()
AQI = AQI.sort_values('Date', ascending = False)
AQIbig6list=['so2','pm10','pm25','co','no2','o3']
AQI_big6df= AQI[AQI['Specie'].isin(AQIbig6list)]





# AQI_average_values = pd.pivot_table(AQI_big6df, index = ['City'], values = 'median', columns = 'Specie',aggfunc = 'mean')
# AQI_average_values_big6 = AQI_average_values[AQI_average_values['so2','pm10','pm25','co','no2','03']]
# AQI_average_values_big6
# aqigroup=['City','Specie']
# AQI_average_values = AQI_big6df.groupby(by = aqigroup).mean()
# AQI.to_csv('AQI.csv')

In [18]:
AQI_big6df

Date Country        City Specie  count   min   max  median  \
373    2021-01-21      US   Manhattan   pm25     43   1.0  59.0    21.0   
377    2021-01-21      US       Miami   pm10     54   1.0  32.0     6.0   
378    2021-01-21      US       Miami     o3     44   0.5  26.4     5.3   
381    2021-01-21      US       Miami     co     48   1.9   3.8     2.5   
367    2021-01-21      US   Manhattan    no2     16  13.0  35.2    18.6   
...           ...     ...         ...    ...    ...   ...   ...     ...   
212102 2019-12-30      US     Atlanta     o3     22  18.4  28.9    24.1   
212103 2019-12-30      US     Atlanta    no2     24   1.0   9.4     3.8   
212106 2019-12-30      US     Atlanta   pm25     44   1.0  30.0    15.0   
212108 2019-12-30      US     Atlanta     co     23   1.4   1.9     1.7   
212245 2019-12-30      US  Fort Worth     o3     23   4.0  31.2    18.4   

        variance  
373      1314.67  
377       462.54  
378       824.59  
381         2.59  
367       629.29  
...          ...  
212102    112.43  
212103     48.27  
212106    431.44  
212108      0.15  
212245    526.82  

[93237 rows x 9 columns]

# Merging AQI with  city

Problem to overcome in the merge is that the AQI data geolocates through country and city while the covid_asthma dataset geolocates through state and county. How can we make a new column with only county or city or state for the dataset that lacks it?

We want to create a function that will sort the cities into the state

In [19]:
us_cities = pd.read_csv('uscities.csv')
us_cities['City'] = us_cities['city']
us_cities.head()

city   city_ascii state_id  state_name  county_fips  county_name  \
0     New York     New York       NY    New York        36061     New York   
1  Los Angeles  Los Angeles       CA  California         6037  Los Angeles   
2      Chicago      Chicago       IL    Illinois        17031         Cook   
3        Miami        Miami       FL     Florida        12086   Miami-Dade   
4       Dallas       Dallas       TX       Texas        48113       Dallas   

       lat       lng  population  density   source  military  incorporated  \
0  40.6943  -73.9249    18713220  10715.0  polygon     False          True   
1  34.1139 -118.4068    12750807   3276.0  polygon     False          True   
2  41.8373  -87.6862     8604203   4574.0  polygon     False          True   
3  25.7839  -80.2102     6445545   5019.0  polygon     False          True   
4  32.7936  -96.7662     5743938   1526.0  polygon     False          True   

              timezone  ranking  \
0     America/New_York        1   
1  America/Los_Angeles        1   
2      America/Chicago        1   
3     America/New_York        1   
4      America/Chicago        1   

                                                zips          id         City  
0  11229 11226 11225 11224 11222 11221 11220 1138...  1840034016     New York  
1  90291 90293 90292 91316 91311 90037 90031 9000...  1840020491  Los Angeles  
2  60018 60649 60641 60640 60643 60642 60645 6064...  1840000494      Chicago  
3  33129 33125 33126 33127 33128 33149 33144 3314...  1840015149        Miami  
4  75287 75098 75233 75254 75251 75252 75253 7503...  1840019440       Dallas

In [23]:
aqi_city = pd.merge(AQI_big6df,us_cities,
    how="left",                      
    left_on = ['City'],
    right_on = ['City']) 
aqi_city.head()

Date Country       City Specie  count  min   max  median  variance  \
0 2021-01-21      US  Manhattan   pm25     43  1.0  59.0    21.0   1314.67   
1 2021-01-21      US  Manhattan   pm25     43  1.0  59.0    21.0   1314.67   
2 2021-01-21      US  Manhattan   pm25     43  1.0  59.0    21.0   1314.67   
3 2021-01-21      US  Manhattan   pm25     43  1.0  59.0    21.0   1314.67   
4 2021-01-21      US      Miami   pm10     54  1.0  32.0     6.0    462.54   

        city  ...       lng population  density   source military  \
0  Manhattan  ...  -73.9662  1628706.0  27545.0  polygon    False   
1  Manhattan  ...  -96.6046    57046.0   1062.0  polygon    False   
2  Manhattan  ...  -87.9805     8240.0    476.0  polygon    False   
3  Manhattan  ... -111.3343     1906.0    421.0  polygon    False   
4      Miami  ...  -80.2102  6445545.0   5019.0  polygon    False   

   incorporated          timezone  ranking  \
0          True  America/New_York      1.0   
1          True   America/Chicago      3.0   
2          True   America/Chicago      3.0   
3          True    America/Denver      3.0   
4          True  America/New_York      1.0   

                                                zips            id  
0  10026 10027 10030 10037 10039 10001 10011 1001...  1.840034e+09  
1                            66506 66502 66503 66505  1.840002e+09  
2                                              60442  1.840011e+09  
3                                              59741  1.840022e+09  
4  33129 33125 33126 33127 33128 33149 33144 3314...  1.840015e+09  

[5 rows x 26 columns]

In [24]:
'''
dropping columns and renaming for clarity before merge
'''
aqicitydrop = ['source', 'military', 'incorporated', 'timezone','lat', 'lng','county_fips', 'Country', 'City', 'ranking', 'zips','id','state_id', 'state_name','city_ascii']
aqi_city = aqi_city.drop(columns= aqicitydrop)
aqi_city = aqi_city.rename(columns={'state_name':'state','county_name':'county'})


# Merging Covid/Asthma/AQI

In [25]:
aqi_city

Date Specie  count  min   max  median  variance        city  \
0      2021-01-21   pm25     43  1.0  59.0    21.0   1314.67   Manhattan   
1      2021-01-21   pm25     43  1.0  59.0    21.0   1314.67   Manhattan   
2      2021-01-21   pm25     43  1.0  59.0    21.0   1314.67   Manhattan   
3      2021-01-21   pm25     43  1.0  59.0    21.0   1314.67   Manhattan   
4      2021-01-21   pm10     54  1.0  32.0     6.0    462.54       Miami   
...           ...    ...    ...  ...   ...     ...       ...         ...   
467037 2019-12-30     co     23  1.4   1.9     1.7      0.15     Atlanta   
467038 2019-12-30     co     23  1.4   1.9     1.7      0.15     Atlanta   
467039 2019-12-30     co     23  1.4   1.9     1.7      0.15     Atlanta   
467040 2019-12-30     co     23  1.4   1.9     1.7      0.15     Atlanta   
467041 2019-12-30     o3     23  4.0  31.2    18.4    526.82  Fort Worth   

            county  population  density  
0         New York   1628706.0  27545.0  
1            Riley     57046.0   1062.0  
2             Will      8240.0    476.0  
3         Gallatin      1906.0    421.0  
4       Miami-Dade   6445545.0   5019.0  
...            ...         ...      ...  
467037       Macon       380.0    428.0  
467038      Cowley       181.0    146.0  
467039        Winn       146.0     52.0  
467040      Phelps       132.0    220.0  
467041     Tarrant    909585.0   1016.0  

[467042 rows x 11 columns]

In [26]:
master1 = pd.merge(covid_asthma,aqi_city,
    how="left",                      
    left_on = ['county'],
    right_on = ['county']
)
master1

MemoryError: Unable to allocate 5.62 GiB for an array with shape (753807573,) and data type int64

Ideas on kernel death:
1. if joining by state, different cities will be joined together? --> refine to match by city or county instead of state probably by county

2.  5.62 GiB for array with shape 753807573


# HistAQI


In [ ]:
# histAQI = pd.read_excel('airqualitytrendsbycity1990-2019.xlsx')
# histAQI.head()

In [ ]:
histAQI_Year_column = pd.melt(histAQI, id_vars=(['Pollutant','StateAbbr','Trend Statistic']), value_vars=([1990,
                                1991,                          1992,
                                1993,                          1994,
                                1995,                          1996,
                                1997,                          1998,
                                1999,                          2000,
                                2001,                          2002,
                                2003,                          2004,
                                2005,                          2006,
                                2007,                          2008,
                                2009,                          2010,
                                2011,                          2012,
                                2013,                          2014,
                                2015,                          2016,
                                2017,                          2018,
                                2019]), var_name='Year', value_name='Pollutant Value').dropna()
histAQI_Year_column.head()

In [ ]:
cities = histAQI['Core Based Statistical Area']
y = [str(i).split() for i in cities]

states = []
for i in y:
    state = i[-1]
    states.append(state)
histAQI['StateAbbr'] = states

In [ ]:
histAQI.head()

In [ ]:
histAQI_asthma= histAQI_asthma[histAQI_asthma.StateAbbr != 'US']

In [ ]:
histAQI_asthma

In [ ]:
#copied code to fill in values from histAQI_asthma
sns.lineplot(x = "Year", y = "Pollutant Value", data = histAQI_Year_column)
plt.title('Total Pollutant Values over Time')
plt.show()

In [ ]:
sns.lineplot(x = "Year", y = "Pollutant Value", data = histAQI_Year_column)
plt.title('Total Pollutant Values over Time')
plt.show()

In [ ]:
sns.lineplot(x = "Year", y = "Pollutant Value", data = histAQI_Year_column, hue = "Pollutant")
plt.title('Pollutant Values over Time')
plt.show()

# Merge AQI and Asthma

In [32]:
#append adds rows
# histAQI_asthma = histAQI_Year_column.append(asthma_premerge)

#merge combines
AQI_asthma = pd.merge(asthma1,AQI_big6df,
    how="left",                      
    left_on = ['state'],
    right_on = ['state'])                        
AQI_asthma


KeyError: 'state'

# Demographics


In [ ]:
demographics_data = pd.read_csv("https://data.cdc.gov/resource/vbim-akqf.csv")

In [ ]:
demographics_data.groupby('race_ethnicity_combined').size()


In [ ]:
demographics_data.describe()

Date Country             City       Specie  count     min     max  \
612567  9/9/2020      US         Hartford         pm10     48    10.0    32.0   
655525  9/9/2020      US      Springfield           o3     24     1.6    32.8   
622276  9/9/2020      US          Houston          no2     93     0.3    23.3   
587151  9/9/2020      US         Portland     pressure    188    29.5  1017.9   
636160  9/9/2020      US        Charlotte         pm25     47     5.0    59.0   
611427  9/9/2020      US         Hartford     humidity     48    48.0    91.7   
703393  9/9/2020      US            Miami    wind-gust     55     0.2    11.0   
576690  9/9/2020      US           Boston         pm25    133     5.0    65.0   
622961  9/9/2020      US          Houston     pressure    114  1010.2  1018.0   
607360  9/9/2020      US        Milwaukee         pm10    119     1.0     8.0   
606610  9/9/2020      US        Milwaukee          dew     28    10.0    13.0   
573099  9/9/2020      US          Jackson          no2     22     1.0     9.1   
765115  9/9/2020      US           Queens           o3     69     0.8    35.2   
617639  9/9/2020      US          Chicago   wind-speed    206     0.1    10.3   
723260  9/9/2020      US        San Diego   wind-speed     15     0.2     6.5   
621816  9/9/2020      US          Houston           o3    118     0.8    38.4   
641631  9/9/2020      US          Oakland          no2    122     1.9    19.5   
715396  9/9/2020      US        Las Vegas    wind-gust    148     0.1    20.1   
718712  9/9/2020      US      San Antonio  temperature    110    17.3    30.5   
639749  9/9/2020      US       Sacramento     humidity     62    23.6    74.5   
606558  9/9/2020      US        Milwaukee     pressure    167  1019.8  1032.1   
727389  9/9/2020      US         Columbus          dew     23    17.0    20.0   
675778  9/9/2020      US      Little Rock   wind-speed     46     1.5     5.1   
619253  9/9/2020      US          Chicago  temperature    215     8.6    19.0   
708818  9/9/2020      US     Jacksonville   wind-speed     67     0.1     4.1   
735394  9/9/2020      US           Denver  temperature    209    -0.6     8.0   
766005  9/9/2020      US           Queens     pressure     96  1022.9  1026.7   
745504  9/9/2020      US      Albuquerque     pressure     72  1010.0  1027.1   
632196  9/9/2020      US          Atlanta   wind-speed     15     0.1     1.5   
592796  9/9/2020      US           Austin     humidity     54    86.7    97.0   
...          ...     ...              ...          ...    ...     ...     ...   
610186  1/1/2020      US         Hartford         pm25     47     9.0    32.0   
634897  1/1/2020      US        Charlotte           o3     46     2.4    32.8   
762507  1/1/2020      US           Tucson          no2     24     2.9    19.3   
677220  1/1/2020      US        Baltimore          so2      2     1.5     1.5   
561504  1/1/2020      US    Oklahoma City         pm10     96     4.0    18.0   
616371  1/1/2020      US  Washington D.C.          so2     23     0.2     0.6   
656861  1/1/2020      US      Springfield   wind speed      8     2.1     4.0   
745679  1/1/2020      US    San Francisco          no2     68     1.0    22.3   
652201  1/1/2020      US         Brooklyn         pm25    168     1.0    54.0   
568132  1/1/2020      US          Raleigh         pm10     24     2.0    21.0   
626120  1/1/2020      US     Indianapolis          no2     22     5.6    10.2   
721224  1/1/2020      US     Philadelphia    wind-gust     22     1.6    10.8   
593219  1/1/2020      US           Austin         pm25     48     8.0   127.0   
720255  1/1/2020      US     Philadelphia           co     20     1.2    10.0   
725437  1/1/2020      US         Columbus         pm25     24     4.0    35.0   
688433  1/1/2020      US          El Paso           o3    166     0.8    35.2   
680905  1/1/2020      US          Phoenix         pm10    190     1.0   473.0   
641618  1/1/2020      US          Oakland  